app/views/checklist_tests/_checklist_measure.html.erb

```diff
@@ -53,11 +60,15 @@
                   <td/>
                 </tr>
                 <% else %>
-                <td rowspan="3"><%= criteria[:description].chomp(': ' + criteria[:title]) %></td>
+                <td rowspan="3" class='hide-me'><%= criteria[:description].chomp(': ' + criteria[:title]) %></td>
+                <%end%>
```

Comment:
Why was this end moved here from line 85.  This is causing a strange display for transfers.

![screen shot 2017-06-07 at 11 24 20 am](https://user-images.githubusercontent.com/8173551/26886634-e5e435a8-4b73-11e7-84ed-55cab85af7d0.png)

It should look like this

![screen shot 2017-06-07 at 11 25 58 am](https://user-images.githubusercontent.com/8173551/26886723-1ed46306-4b74-11e7-81c7-fed4aff64098.png)